packages/card/src/card.ts

```diff
@@ -104,4 +119,10 @@ export class Card extends LitElement {
                 return this.renderDefault();
         }
     }
+
+    protected firstUpdated(changes: PropertyValues): void {
+        super.firstUpdated(changes);
+        this.setAttribute('role', 'figure');
```

Comment:
I don't agree with role=figure for a card. We have no idea what is going to be in the card so figure seems wrong. If we are going to generalize to something it should be role=region or group depending if we want it to be landmark of not (I vote no in most cases so would lean towards group) but we also need to allow it to be overridden which it doesn't seem is allowed here
The card should probably also have aria-labelledby referencing the card's title (if it has a role)

Reply:
This is the role currently outlined in the Spectrum CSS metadata for this pattern, which is applied without an `aria-labelledby`, if that's something you think should be happening, you should definitely submit an issue against: https://github.com/adobe/spectrum-css/blob/master/components/card/metadata/card-asset.yml#L8 as altering it would just be a symptom of a larger issue.

Reply:
I'll do that. We don't add figure in react-spectrum.

Reply:
filed https://github.com/adobe/spectrum-css/issues/706